<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBDialog</class>
 <widget class="QWidget" name="DBDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Open database file</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="4">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <item>
      <widget class="ClickableListView" name="dbListView">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Filters</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Player 2:</string>
          </property>
          <property name="buddy">
           <cstring>p2Edit</cstring>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="eventEdit_2">
          <property name="text">
           <string>Event:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>From:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="fromEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Player 1:</string>
          </property>
          <property name="buddy">
           <cstring>p1Edit</cstring>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>To:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="toEdit"/>
        </item>
        <item row="2" column="1" colspan="3">
         <widget class="QLineEdit" name="eventEdit"/>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QLineEdit" name="p2Edit"/>
        </item>
        <item row="0" column="1" colspan="3">
         <widget class="QLineEdit" name="p1Edit"/>
        </item>
        <item row="6" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="gameNumLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>1</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="resetButton">
            <property name="text">
             <string>Reset Games</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="applyButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="dbConfButton">
         <property name="text">
          <string>Configure database &amp;paths...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Open</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="overwriteSGFEncoding">
       <property name="title">
        <string>Override character set for text in SGF file:</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <property name="bottomMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QComboBox" name="encodingList">
          <item>
           <property name="text">
            <string>UTF-8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GB18030</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EUC-KR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EUC-JP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ISO-2022-JP</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Shift-JIS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Big5</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="FigureView" name="boardView">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>File properties</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="5" column="1">
       <widget class="QLabel" name="File_Size">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>19</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_19">
        <property name="text">
         <string>Komi:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLabel" name="File_Date">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
      </item>
      <item row="5" column="5">
       <widget class="QLabel" name="File_Handicap">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLabel" name="label_20">
        <property name="text">
         <string>Handicap:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Date:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLabel" name="File_Komi">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>6.5</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLabel" name="File_Result">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="q4go.qrc">:/images/stone_b16.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Result:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="5">
       <widget class="QLabel" name="File_BlackPlayer">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5">
       <widget class="QLabel" name="File_WhitePlayer">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>White</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="q4go.qrc">:/images/stone_w16.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Event:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLabel" name="File_Event">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Round:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QLabel" name="File_Round">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="_7">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="goFirstButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Go to first move.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="q4go.qrc">
            <normaloff>:/BoardWindow/images/boardwindow/2leftarrow.png</normaloff>:/BoardWindow/images/boardwindow/2leftarrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="goPrevButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Go to previous move.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="q4go.qrc">
            <normaloff>:/BoardWindow/images/boardwindow/1leftarrow.png</normaloff>:/BoardWindow/images/boardwindow/1leftarrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="goNextButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Go to next move.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="q4go.qrc">
            <normaloff>:/BoardWindow/images/boardwindow/1rightarrow.png</normaloff>:/BoardWindow/images/boardwindow/1rightarrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="goLastButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Go to last move.</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="q4go.qrc">
            <normaloff>:/BoardWindow/images/boardwindow/2rightarrow.png</normaloff>:/BoardWindow/images/boardwindow/2rightarrow.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FigureView</class>
   <extends>QGraphicsView</extends>
   <header>board.h</header>
  </customwidget>
  <customwidget>
   <class>ClickableListView</class>
   <extends>QListView</extends>
   <header>clickableviews.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>dbListView</tabstop>
  <tabstop>p1Edit</tabstop>
  <tabstop>p2Edit</tabstop>
  <tabstop>eventEdit</tabstop>
  <tabstop>fromEdit</tabstop>
  <tabstop>toEdit</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>applyButton</tabstop>
  <tabstop>dbConfButton</tabstop>
  <tabstop>overwriteSGFEncoding</tabstop>
  <tabstop>encodingList</tabstop>
  <tabstop>boardView</tabstop>
 </tabstops>
 <resources>
  <include location="q4go.qrc"/>
 </resources>
 <connections/>
</ui>
